docs: Oryx compositions (#1910)
spryker/spryker-docs: docs/scos/dev/front-end-development/202212.0/oryx/oryx-presets.md

  ...
```

> Feature sets also contain static experience data. Experience data includes the structure and layout of the components of and application, such as pages and sections. By utilizing the static experience data provided by the presets, you don't need to set up any boilerplate code. Moreover, we avoid hardcoded page structures which enables personalized experiences.

Reviewer: ```suggestion
Feature sets also contain static experience data. Experience data includes the structure and layout of the components of an application, such as pages and sections. By utilizing the static experience data provided by the presets, you don't need to set up any boilerplate code. Moreover, we avoid hardcoded page structures that enable personalized experiences.
```

For more information about feature sets, see [Feature sets](/docs/scos/dev/front-end-development/{{page.version}}/oryx/oryx-feature-sets.html)

## Themes

> A theme represents the global visual appearance of an application, including typography, colors, and other specific design elements, such as form field placeholder color. The theme is build with design tokens, which are configurable values that are used in CSS properties. By configuring these design tokens, you can customize the theme and align the application and it's components with your brand identity or specific design requirements.

Reviewer 2: Is this a list? `form field placeholder color`

Author: i've rephrased it

> A *theme* represents the global visual appearance of an application, including typography, colors, and other specific design elements, such as the color of a form field's placeholder. Themes are  built with design tokens, which are configurable values that are used in CSS properties. By configuring these design tokens, you can customize a theme and align the application and it's components with your brand identity or specific design requirements.

Reviewer: ```suggestion
A *theme* represents the global visual appearance of an application, including typography, colors, and other specific design elements, such as the color of a form field's placeholder. Themes are built with design tokens, which are configurable values that are used in CSS properties. Configuring these design tokens lets you customize a theme and align the application and its components with your brand identity or specific design requirements.
```

  ...
```

Feature sets also contain static experience data. Experience data includes the structure and layout of the components of the application, such as pages and sections. By utilizing the static experience data provided by the presets, you do no need to set up any boilerplate code. Moreover, we avoid hardcoded page structures that would not allow for personalized experiences going forward.

For more information about feature sets, see [Feature sets](/docs/scos/dev/front-end-development/{{page.version}}/oryx/oryx-feature-sets.html)

## Themes

Themes in Oryx play a key role in defining the visual appearance of your application. A theme represents the global typography, colors, and other specific design elements, such as form field placeholder color, that define the overall look and feel of your application.

> The theme consists of design tokens, which are configuration values that control the visual aspects of your application. Design tokens allow you to define global typography, colors, spacing, and other visual properties. By configuring these design tokens, you can customize the visual appearance of your application to align with your brand or specific design requirements.

Reviewer 3: In those two paragraphs we are mentioning almost the same thing three times:
- "that define the overall look and feel of your application"
- "that control the visual aspects of your application"
- "customize the visual appearance of your application ..."

Not saying it's necesairly bad, but a bit noisy IMO.

  .create();
```

> Themes play a crucial role in maintaining a consistent and coherent visual experience throughout your application. They provide a centralized way to manage and apply design tokens, ensuring a unified look and feel across components and screens.

Reviewer: Looks like marketing slang
```suggestion
Themes help maintain a consistent and coherent visual experience throughout your application. They provide a centralized way to manage and apply design tokens, ensuring a unified look and feel across components and screens.
```

  .create();
```

> By utilizing themes and design tokens in Oryx presets, you have the flexibility to customize the visual aspects of your application, making it unique and aligned with your branding or specific design preferences.

Reviewer 3: another one
- "customize the visual aspects of your application"

Themes play a crucial role in maintaining a consistent and coherent visual experience throughout your application. They provide a centralized way to manage and apply design tokens, ensuring a unified look and feel across components and screens.

> Take advantage of the themes provided in Oryx presets or create your own themes to tailor the visual appearance of your application according to your specific needs.

Reviewer 3: one more :) 
- ailor the visual appearance of your application
